README.md

```diff
@@ -746,6 +746,22 @@ Check whether the DSS switched buckets. If so, the lambda policy may need to be
 updated to reflect that change. To fix this, redeploy the lambdas (`make 
 package`) in the affected deployment.
 
+## `make requirements_update` does not update transitive requirements
+
+In some cases, `make requirements_update` might not produce any updates to
+transitive requirements, even if you expect them. For example, a sandbox build
+might identify updated transitive requirements even though doing `make
+requirements_update` locally doesn't.
+
+This is a side effect of the Docker build cache on two different machines
+diverging to reflect different states on PyPI. This can be fixed by running
+`make requirements_update` with caching disabled for the requirements generation
+step:
+
+```console
```

Comment:
For the sake of consistency, I'd leave `console` off. It doesn't buy as anything, if we go with the _force target because there is nothing left to syntax-highlight. I'd welcome one commit that changes all of the preformatted blocks to use this where appropriate.

Reply:
Just to be clear, when you say

> I'd welcome one commit that changes all of the preformatted blocks to use this where appropriate.

are you instructing me to

* Adjust all preformatted blocks to ensure there is no explicit syntax-highlighting directive

* Adjust all preformatted blocks to remove the `console` syntax-highlighting directive where it exists

* Adjust all preformatted blocks to add syntax-highlighting directives where appropriate

I am assuming the first option, but I just wanted to double check to make sure.

Reply:
To be precise, I was only asking you to remove `console` here. If you WANT, you can add a separate commit that adds syntax-highlighting to all pre-formatted blocks in this file but there is no "instruction" to do so.

Reply:
Okay, thanks. I'm happy to make that adjustment.

Reply:
In fact, I changed my mind. I just looked at the file. There are so many preformatted blocks with short commands where syntax-highlighting would not add any clarity or ease of use. So don't do that separate commit, only remove `console` here to be consistent with the rest of the file.

Reply:
Ah, I'm sorry -- I didn't see your comment where you said

> But don't feel obliged.

My apologies.

Reply:
No problem.